tests/Oryx.RuntimeImage.Tests/Oryx.RuntimeImage.Tests.csproj

```diff
@@ -18,7 +18,7 @@
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.4.1" />
     <PackageReference Include="xunit" Version="2.4.2" />
     <PackageReference Include="Xunit.SkippableFact" Version="1.4.13" />
-    <PackageReference Include="XunitXml.TestLogger" Version="3.0.70" />
+    <PackageReference Include="XunitXml.TestLogger" Version="3.1.11" />
```

Comment:
Should we also update `XunitXml.TestLogger` in the other places too?
![image](https://github.com/microsoft/Oryx/assets/86327553/6d756423-3839-4e71-9f5a-4bba0c4ed7c2)

Reply:
Hi @waliMSFT I noticed you updated another the version for another `XunitXml.TestLogger` in your [recent commit](https://github.com/microsoft/Oryx/pull/2088/commits/692ccbdff41cf95d6603d7a98b0de0c2dbdb005a). But I see total of 5 files being changed. The screenshot above shows there are 7. Do we not need to update the other 2?

Reply:
I forgot to reply on your comment.
I probably missed the other files. I have updated the package on all the files now.